.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -23,7 +40,6 @@ body:
         - C++
         - C++ - Gandiva
         - C++ - Plasma
-        - Compute IR
         - Continuous Integration
         - Dart
```

Comment:
What is "Dart"?

Reply:
Probably https://dart.dev/

Reply:
We have a single, closed JIRA with this (https://issues.apache.org/jira/browse/ARROW-2631), so probably not worth to have a component for